src/ring/ring_okta/predicates.clj

```diff
@@ -1,5 +1,6 @@
 (ns ring.ring-okta.predicates
-  (:require [ring.util.request :as ring-request]))
+  (:require [ring.util.request :as ring-request]
+            [clojure.string :as str]))
```

Comment:
Looks like this is unused? If so, could we remove the extra import to keep the ns clean?

Reply:
Sure. Will do.

Reply:
Done!